### DISCUSSION TITLE
fit_predict not working with Louvain algorithm

### DISCUSSION THREAD

**Discussion by ADN8B** [sknetwork-team/scikit-network#555] [Q&A]
Hello,

Here is my code :

louvain=Louvain(M)
louvain.fit_predict(M)

But I got the following error : "AttributeError: 'Louvain' object has no attribute 'fit_predict' ".

I searched in the file "louvain.py" but I couldn't find the method "fit_predict".

Does anyone know how to fix it?

**Comment by tbonald** [comment]
Hello,
Which version of scikit-network do you use?
The fit_predict method of Louvain is available in the last version (0.29.0).

**Reply by ADN8B** [reply]
It's working now.
Thank you !
